eng/source-build-tarball-build.yml

```diff
@@ -38,3 +38,10 @@ stages:
     - template: /src/SourceBuild/Arcade/eng/common/templates/steps/source-build-run-tarball-build.yml
       parameters:
         sourceFolder: $(_TarballDir)
+
```

Comment:
Another friendly pointer, if this is move to [here](https://github.com/dotnet/installer/blob/main/src/SourceBuild/Arcade/eng/common/templates/steps/source-build-run-tarball-build.yml), it will get run in both the PR validation (only when source-build src is changed) and internal builds.  Where this is located now, it will only run in the internal builds.